<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerControls</class>
 <widget class="QWidget" name="ViewerControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="mViewTypeTabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m2DTab">
      <attribute name="title">
       <string>2D</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Density</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBoxLUTWindow">
             <property name="title">
              <string>Window</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Center</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="windowCenterSpinBox">
                  <property name="minimum">
                   <number>-4000</number>
                  </property>
                  <property name="maximum">
                   <number>4000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Width</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="windowWidthSpinBox">
                  <property name="minimum">
                   <number>-4000</number>
                  </property>
                  <property name="maximum">
                   <number>4000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTilingSetup">
         <property name="title">
          <string>Viewport tiling</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Rows</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="viewportTilesRows">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Columns</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="viewportTilesCols">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>15</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Slice step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sliceStep">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m3DTab">
      <attribute name="title">
       <string>3D</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTabWidget" name="tabWidget_3">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Transfer function</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="mEnablePreintegratedTFCheckBox">
             <property name="text">
              <string>Preintegrated transfer  function</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>MIP</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>Density</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_7">
          <attribute name="title">
           <string>Isosurface</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Isovalue:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="mIsoValueSlider">
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QDoubleSpinBox" name="mIsoValueSpinBox">
             <property name="maximum">
              <double>5000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="volumeRestrictionsGroupBox">
         <property name="title">
          <string>View volume restrictions</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X interval</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="xIntervalASpinBox">
            <property name="minimum">
             <double>-2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="xIntervalBSpinBox">
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Y interval</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="yIntervalASpinBox">
            <property name="minimum">
             <double>-2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="yIntervalBSpinBox">
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Z interval</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="zIntervalASpinBox">
            <property name="minimum">
             <double>-2.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="zIntervalBSpinBox">
            <property name="maximum">
             <double>3.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QPushButton" name="volumeRestrictionsResetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Cut planes</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>xIntervalASpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yIntervalASpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zIntervalASpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xIntervalBSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yIntervalBSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zIntervalBSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>windowCenterSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>windowWidthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeRestrictionsGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeRestrictionsResetButton</sender>
   <signal>clicked()</signal>
   <receiver>ViewerControls</receiver>
   <slot>resetVolumeRestrictions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewportTilesRows</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewportTilesCols</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceStep</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mViewTypeTabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>12</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mIsoValueSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ViewerControls</receiver>
   <slot>settingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>settingsChanged()</slot>
  <slot>resetVolumeRestrictions()</slot>
 </slots>
</ui>
